app/viewmodels/govuk/RadiosFluency.scala

```diff
@@ -59,12 +59,12 @@ trait RadiosFluency {
              )(implicit messages: Messages): Radios =
       yesNo(
         field    = field,
-        fieldset = FieldsetViewModel(legend)
+        fieldset = FieldsetViewModel(legend),
```

Comment:
You might need to remove the trailing commas here to avoid issues in the pipeline